### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by rhagarty**
- updated readme
- add journey docs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -253,21 +297,20 @@ INFO: Checking pre-reqs for running TPC-DS queries is successful.
...
-### Option (Q) - Quit
-This will exit the script
+#### Cleanup option: "(Q) - Quit"
 
-## Running using Notebook.
-### Steps
+This will exit the script.
 
-Follow these steps to setup and run this developer journey. These steps are described in detail below.
+## Run using a Jupyter notebook in the IBM Data Science Experience (DSX) on IBM Bluemix
```
Comment (stevemart) [inline README.md]: I'd shorten this a pinch to "Run using a Jupyter notebook in the IBM Data Science Experience"

File: README.md [hunk 2]
```diff
@@ -1,60 +1,83 @@
...
-#### Development Tools
-Make sure the required development tools are installed in your platform. This journey is supported on Mac and Linux
-platforms only.
+``` 
+git clone https://github.com/spark-tpc-ds-performance-test 
+```
+
+### 2. Setup development tools
+
+Make sure the required development tools are installed in your platform. This journey is supported on Mac and Linux platforms only.
```
Comment (stevemart) [inline README.md]: tack on a bit at the end: "This journey is supported on Mac and Linux platforms only, depending on your platform, run the following command to install the necessary development tools"

File: README.md [hunk 3]
```diff
@@ -1,60 +1,83 @@
...
 * **Ubuntu:** ``` sudo apt-get install gcc make flex bison byacc git ```
 * **CentOS/RHEL:** ``` sudo yum install gcc make flex bison byacc git ```
 * **MacOS:** ``` xcode-select --install ```
```
Comment (stevemart) [inline README.md]: add a `$` to each of these to indicate it's running in a terminal

**Comment by rhagarty** [comment]
Update with Steve's corrections

**Comment by dilipbiswal** [comment]
@rhagarty Can you please push your changes. I would like to put the arch-diagram after rebasing with your changes. Thanks for making the edits.

**Comment by stevemart** [comment]
@dilipbiswal done

**Comment by dilipbiswal** [comment]
Thank you @stevemart

**Pull request merged**